bindings/pyroot_experimental/pythonizations/test/numbadeclare.py

```diff
@@ -20,14 +37,17 @@ class NumbaDeclareSimple(unittest.TestCase):
 
     test_inputs = default_test_inputs
 
+
     # Test refcounts
+    @unittest.skipIf(skip, skip_condition)
```

Comment:
There are two issues, first we never call `unittest.main()` for our tests. The pyunittest framework scans for `unittest.TestCase` classes and runs them on his own. Second, we cannot just return via sys exit (`sys.exit(0)`) because this is detected by pyunittest as a failure.

Actually, we can put a huge `if not skip: class TestClass(unittest.TestCase): ...` around everything. Then we don't tell the test system that we actively skip but we have just an empty test file, which is counted as success.

Reply:
I merged to get the nightlies. Though if you think we should not annotate but use another mechanic to skip the tests, feel free to tell me. Most likely we'll replicate the mechanics used in the numba integration to other python related features, so we should be fine with the approach here!